packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java

```diff
@@ -135,6 +156,29 @@ public void onImageReady(List<Image> images) {
     return false;
   }
 
+  @Override
+  public boolean onRequestPermissionsResult(
+      int requestCode, String[] permissions, int[] grantResults) {
+    if (requestCode == REQUEST_PERMISSIONS) {
+      if (grantResults.length == 2
+          && grantResults[0] == PackageManager.PERMISSION_GRANTED
+          && grantResults[1] == PackageManager.PERMISSION_GRANTED) {
+        Activity activity = registrar.activity();
+        if (activity == null) {
+          pendingResult.error(
+              "no_activity", "image_picker plugin requires a foreground activity.", null);
+        }
+        activity.startActivityForResult(
+            cameraModule.getCameraIntent(activity), REQUEST_CODE_CAMERA);
+      } else {
+        pendingResult = null;
```

Comment:
Should this send an error back to the Dart side if the permissions were not granted?

Reply:
Done - also made sure the example was handling the error